x-pack/plugins/observability_solution/apm/public/components/routing/templates/apm_main_template/index.tsx

```diff
@@ -101,6 +100,8 @@ export function ApmMainTemplate({
     location.pathname.includes(path)
   );
 
+  const isEntityExperienceFullyEnabled = isEntityCentricExperienceEnabled && isEntityManagerEnabled;
```

Comment:
Why do we need to care if isEntityManagerEnabled is enabled?

Only looking at the FF and the serviceEntitySummary?.dataStreamTypes is enough to bypass the no data screen. We know serviceEntitySummary?.dataStreamTypes will only be populated if EEM is enabled.

Reply:
Oh, got it! I thought it was only done that way in Infra.

I’ll go ahead and remove the dependency and use the Advanced Setting along with the dataStreamTypes.

Thanks @cauemarcondes!

Reply:
Done in https://github.com/elastic/kibana/pull/196387/commits/1fbf28db41eddd17885bf6a6c00018342a14515b